spec/share/asciidoctor.spec.js

```diff
@@ -529,6 +529,27 @@ var shareSpec = function (testOptions, asciidoctor) {
         expect(html).toContain('March');
       });
     });
+
+    describe('Converter', function () {
```

Comment:
Before knowing about the subtleties of the arrow functions I was using them everywhere in the tests suite.
Mocha explicitly discouraged the usage of arrow functions: https://mochajs.org/#arrow-functions

I think we should follow this rule and always use `function` to be consistent.

Reply:
I strongly disagree on this one. The arrow functions make JavaScript 1,000 times easier to read. The function should only be used when you need `this`. In fact, it's a very useful way of suppressing the `this` context change.

That recommendation from mocha is out of date. They actually recommend the opposite now.

(of course, until we switch the code style, we shouldn't make any sweeping changes).

Reply:
I'm kinda relieved because I like the concise syntax of arrow functions :+1: 
Once bitten, twice shy :flushed:

Reply:
Be bold! :muscle: